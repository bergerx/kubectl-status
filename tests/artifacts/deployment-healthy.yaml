apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    deployment.kubernetes.io/revision: "1"
  creationTimestamp: "2020-03-18T01:24:09Z"
  generation: 1
  labels:
    run: httpbin-deployment
  name: httpbin-deployment
  namespace: test1
  resourceVersion: "311461"
  selfLink: /apis/apps/v1/namespaces/test1/deployments/httpbin-deployment
  uid: 4feef9ce-7e6d-4a28-a973-7ea3c0a49d6f
spec:
  progressDeadlineSeconds: 600
  replicas: 3
  revisionHistoryLimit: 10
  selector:
    matchLabels:
      run: httpbin-deployment
  strategy:
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
    type: RollingUpdate
  template:
    metadata:
      creationTimestamp: null
      labels:
        run: httpbin-deployment
    spec:
      containers:
      - image: kennethreitz/httpbin
        imagePullPolicy: Always
        name: httpbin-deployment
        ports:
        - containerPort: 80
          protocol: TCP
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      terminationGracePeriodSeconds: 30
status:
  availableReplicas: 3
  conditions:
  - lastTransitionTime: "2020-03-18T01:24:49Z"
    lastUpdateTime: "2020-03-18T01:24:49Z"
    message: Deployment has minimum availability.
    reason: MinimumReplicasAvailable
    status: "True"
    type: Available
  - lastTransitionTime: "2020-03-18T01:24:09Z"
    lastUpdateTime: "2020-03-18T01:24:49Z"
    message: ReplicaSet "httpbin-deployment-79f6dfbb9" has successfully progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  observedGeneration: 1
  readyReplicas: 3
  replicas: 3
  updatedReplicas: 3
