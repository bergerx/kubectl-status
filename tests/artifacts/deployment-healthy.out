
Deployment/httpbin-deployment -n test1, created 1m ago, gen:1 rev:1
  Current: Deployment is available. Replicas: 3
  desired:3, existing:3, ready:3, updated:3, available:3
  Available MinimumReplicasAvailable, Deployment has minimum availability. for 1m
  Progressing NewReplicaSetAvailable, ReplicaSet "httpbin-deployment-79f6dfbb9" has successfully progressed. for 1m
